contrib/setup-smt-switch.sh

```diff
@@ -3,7 +3,8 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 DEPS=$DIR/../deps
 
-SMT_SWITCH_VERSION=2e6f51816852b605e06205c37664af882e5cc732
+# TEMP pull branch
+# SMT_SWITCH_VERSION=2e6f51816852b605e06205c37664af882e5cc732
```

Comment:
this is commented. Is it intentional?

Reply:
Yeah it wasn't really necessary, but it's just until the smt-switch branch is merged.